modules/commands/ms_read.cpp

```diff
@@ -167,6 +167,13 @@ class CommandMSRead : public Command
 				for (i = 0, end = mi->memos->size() - 1; i < end; ++i);
 				MemoListCallback::DoRead(source, mi, ci, i);
 			}
+			else if (numstr.equals_ci("ALL"))
+			{
+				for (i = 0; i < mi->memos->size(); i++)
```

Comment:
It may be convention to use ```++i``` here, instead of ```i++```.  I guess that's up to @Adam- though.